### PR TITLE
update release-notes.md [WIP]

### PR DESCRIPTION
**Pull request by SmritiSatya**
Rephrase sentences, changed to bullet points, sentence case
Signed-off-by: Smriti S <smriti.satyanarayana@harness.io>

### CODE REVIEW

**Review by douglas-j-bothwell — APPROVED** [review]
All looks good! Approved.

File: docs/chaos-engineering/introduction/release-notes.md [hunk 1]
```diff
@@ -11,73 +11,104 @@ Harness deploys updates progressively to different Harness cluster hosting accou
...
+* Failing to map a chaos fault to a chaos fault icon doesn’t throw an error on the user interface (Chaos-814)
+	* When you search for a specific chaos fault and the chaos manager cannot map this chaos fault to a chaos fault icon in the backend, it doesn’t throw an error on the user interface. Instead, it silently skips the error logs. 
+
+* Expected Resilience Score doesn’t change when it is overridden (Chaos-791)
+	* The Expected Resilience Score does not change to `NaN` when overridden.
+
+* Resource-type field has been made available (Chaos-714)
+	* The resource-type field was previously not available. Now, it has been made available and you can utilize it to abort a chaos experiment in the audit trail.
+
+* UI/UX issues were fixed
```
Comment (michaelcretzman) [inline docs/chaos-engineering/introduction/release-notes.md:70]: no ticket number?

File: docs/chaos-engineering/introduction/release-notes.md [hunk 2]
```diff
@@ -11,73 +11,104 @@ Harness deploys updates progressively to different Harness cluster hosting accou
...
-1. Chaos infrastructure can't be installed through Harness Delegate.
-2. Logs for chaos infrastructure can’t be viewed.
-3. The properties of chaos infrastructure can’t be updated. You will need to provide blacklisted namespaces.
-4. The properties of the environment to which the chaos infrastructure belongs can’t be updated.
-5. Configuring chaos infrastructure doesn’t provide support for Linux and Windows.
+* Chaos infrastructure cannot be installed through Harness Delegate.
+* Logs for chaos infrastructure cannot be viewed.
+* The properties of chaos infrastructure cannot be updated. You will need to provide blacklisted namespaces.
+* The properties of the environment to which the chaos infrastructure belongs cannot be updated.
+* Configuring chaos infrastructure doesnot provide support for Linux and Windows.
```
Comment (michaelcretzman) [inline docs/chaos-engineering/introduction/release-notes.md:93]: change to "does not"

**Review by michaelcretzman — CHANGES_REQUESTED** [review]
minor things

File: docs/chaos-engineering/introduction/release-notes.md (hunk 1 above)
Comment (SmritiSatya) [reply docs/chaos-engineering/introduction/release-notes.md:70]: No, Michael. There weren't ticket numbers for some of the fixes/what's new.

**Pull request merged**